pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1367,7 +1373,8 @@ def init_arc_param(arctype):
     model.p_sigma_BeneficialReuse = Param(
         model.s_O,
         model.s_T,
-        default=0,
+        # Use a negative number for default so later we can detect for which indexes the user did/did not provide a parameter value
+        default=-1,
```

Comment:
Unfortunately, I don't think this works as expected :( . I tested changing "ReuseCapacity" tab like so:
![image](https://github.com/project-pareto/project-pareto/assets/91741082/47c5d3ef-3644-43a5-a54c-2ad8ece20994)

The 0 values and >0 values work well. The blank cells, do not however. When they are processed by `model.df_parameters["ReuseCapacity"].items()` [line 1384], the blank cells return 0 and is then processed as a capacity of 0. 

I think this is a good future feature. I've added it to the issue.  Maybe, for now, we can remove the " (leave cells blank to indicate infinite capacity)" in the excel files on the ReuseCapacity tabs.